chain/chain/src/runtime/tests.rs

```diff
@@ -369,8 +369,12 @@ impl TestEnv {
     }
 
     pub fn view_account(&self, account_id: &AccountId) -> AccountView {
-        let shard_id =
-            self.epoch_manager.account_id_to_shard_id(account_id, &self.head.epoch_id).unwrap();
+        let shard_id = EpochInfoProvider::account_id_to_shard_id(
```

Comment:
Good question :)

`epoch_manager: Arc<EpochManagerHandle>` implements `EpochInfoProvider` and `EpochManagerAdapter`.
Previously `account_id_to_shard_id` was unique to `EpochInfoProvider`. But in the first commit of this PR, I add the method to `EpochManagerAdapter`, which makes this call ambiguous. Rather than picking a slightly different name, I chose to specify the trait to call here.